packages/larva-patterns/modules/mega-menu-item/mega-menu-item.twig

```diff
@@ -1,9 +1,15 @@
-<li class="mega-menu-item // mega-menu__parent-list-item {{ mega_menu_item_classes }} ">
+<li class="mega-menu-item // mega-menu__parent-list-item {{ mega_menu_item_classes }} " data-collapsible="collapsed" >
 	<div class="mega-menu__item-link // {{ mega_menu_item_link_classes }} ">
```

Comment:
Will this cause any regressions on other brands?

Reply:
No, Verified on RS and this does not create any regression issues.
Future ref: https://penskemediacorp.slack.com/archives/C03RZKELNR5/p1664194743857379?thread_ts=1664187688.768309&cid=C03RZKELNR5